### PR TITLE
Pluralize LinkedDeliveryID

### PR DESCRIPTION
**Pull request by markborkum**
See `proposals/2019/Pluralize LinkedDeliveryID.md` for details.

### CODE REVIEW

**Comment by markborkum** [comment]
@nllong, Ostensibly, this is the last Audit-Template-related pull request for the current sprint.

File: BuildingSync.xsd [hunk 1]
```diff
@@ -11463,6 +11472,7 @@
         </xs:annotation>
         <xs:simpleType>
           <xs:restriction base="xs:string">
+
```
Comment (nllong) [inline BuildingSync.xsd]: @markborkum can you remove this extra line?

Comment (markborkum) [reply BuildingSync.xsd]: @nllong, No. Not because I don't want to. It's not appearing my text editor (Atom).

**Comment by markborkum** [comment]
@nllong, Correct. This is due to the use of the `<xs:choice>` element in the definition of the `<auc:DeliveryType>` element. In Audit Template, there are certain HVAC system configurations that result in both `<auc:CentralAirDistribution>` and `<auc:ZoneEquipment>` being asserted. However, as they cannot be child elements of the same `<auc:DeliveryType>` element, we need to create two `<auc:Delivery>` elements, which are then linked to via one `<auc:OtherHVACSystem>` element.

File: BuildingSync.xsd (hunk 1 above)
Comment (nllong) [reply BuildingSync.xsd]: Hmm, weird, okay. Not sure where this is coming from then...

**Comment by nllong** [comment]
I see. So you can have a radiance ceiling and a VAV box (something similar to a DOAS system) but want to only have one single humidifier defined for both?

**Pull request merged**